package.json

```diff
@@ -2,24 +2,28 @@
   "name": "@c-frame/aframe-particle-system-component",
   "version": "1.1.4",
   "description": "Particle systems for A-Frame.",
-  "main": "index.js",
+  "browser": "index.js",
   "directories": {
     "example": "examples"
   },
   "devDependencies": {
-    "browserify": "^13.0.0",
-    "browserify-css": "^0.9.1",
-    "budo": "^8.2.2",
-    "randomcolor": "^0.4.4",
-    "semistandard": "^8.0.0",
-    "shelljs": "^0.7.0",
-    "shx": "^0.1.1",
-    "snazzy": "^4.0.0",
-    "webpack": "^1.13.0"
+    "browserify": "^17.0.0",
+    "browserify-css": "^0.15.0",
```

Comment:
you can remove browserify and browserify-css

Reply:
done